File: app/src/main/java/com/simplecity/amp_library/ui/views/PlayerViewAdapter.java
package com.simplecity.amp_library.ui.views;


import android.support.annotation.Nullable;

import com.afollestad.materialdialogs.MaterialDialog;
import com.simplecity.amp_library.model.Song;
import com.simplecity.amp_library.tagger.TaggerDialog;

public abstract class PlayerViewAdapter implements PlayerView {

    @Override
    public void setSeekProgress(int progress) {

    }

    @Override
    public void currentTimeVisibilityChanged(boolean visible) {

    }

    @Override
    public void currentTimeChanged(long seconds) {

    }

    @Override
    public void totalTimeChanged(long seconds){

    }

    @Override
    public void queueChanged(int queuePosition, int queueLength) {

    }

    @Override
    public void playbackChanged(boolean isPlaying) {

    }

    @Override
    public void shuffleChanged(int shuffleMode) {

    }

    @Override
    public void repeatChanged(int repeatMode) {

    }

    @Override
    public void favoriteChanged(boolean isFavorite) {

    }

    @Override
    public void trackInfoChanged(@Nullable Song song) {

    }

    @Override
    public void showToast(String message, int duration) {

    }

    @Override
    public void showLyricsDialog(MaterialDialog dialog) {

    }

    @Override
    public void showTaggerDialog(TaggerDialog taggerDialog) {

    }

    @Override
    public void showSongInfoDialog(MaterialDialog dialog) {

    }

}

File: app/src/main/java/com/simplecity/amp_library/ui/views/PlayerViewAdapter.kt
package com.simplecity.amp_library.ui.views

import com.simplecity.amp_library.model.Playlist
import com.simplecity.amp_library.model.Song
import com.simplecity.amp_library.ui.screens.nowplaying.PlayerView

abstract class PlayerViewAdapter : PlayerView {

    override fun setSeekProgress(progress: Int) {

    }

    override fun currentTimeVisibilityChanged(visible: Boolean) {

    }

    override fun currentTimeChanged(seconds: Long) {

    }

    override fun totalTimeChanged(seconds: Long) {

    }

    override fun queueChanged(queuePosition: Int, queueLength: Int) {

    }

    override fun playbackChanged(isPlaying: Boolean) {

    }

    override fun shuffleChanged(shuffleMode: Int) {

    }

    override fun repeatChanged(repeatMode: Int) {

    }

    override fun favoriteChanged(isFavorite: Boolean) {

    }

    override fun trackInfoChanged(song: Song?) {

    }

    override fun showLyricsDialog() {

    }

    override fun showUpgradeDialog() {

    }

    override fun presentCreatePlaylistDialog(songs: List<Song>) {

    }

    override fun presentSongInfoDialog(song: Song) {

    }

    override fun onSongsAddedToPlaylist(playlist: Playlist, numSongs: Int) {

    }

    override fun onSongsAddedToQueue(numSongs: Int) {

    }

    override fun presentTagEditorDialog(song: Song) {

    }

    override fun presentDeleteDialog(songs: List<Song>) {

    }

    override fun shareSong(song: Song) {

    }

    override fun presentRingtonePermissionDialog() {

    }

    override fun showRingtoneSetMessage() {

    }
}